state/package_test.go

```diff
@@ -15,7 +15,7 @@ func TestPackage(t *testing.T) {
 	// At this stage, Juju only supports running the apiservers and database
 	// on Ubuntu. If we end up officially supporting CentOS, then we should
 	// make sure we run the tests there.
-	if os.HostOS() != os.Ubuntu {
+	if false && os.HostOS() != os.Ubuntu {
```

Comment:
it was to force the tests to run on Mac, but yeah, I should pull it out, thanks for catching.